rcgen/src/key_pair.rs

```diff
@@ -352,29 +364,29 @@ impl KeyPair {
 }
 
 #[cfg(feature = "crypto")]
-impl TryFrom<&[u8]> for KeyPair {
+impl TryFrom<&PrivatePkcs8KeyDer<'_>> for KeyPair {
```

Comment:
Why did you remove the old slice conversion functionality? It is easier to go through slices.